packages/api/migrations/20200820002013-create-nomination.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Nominations', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4
+      },
+      status: {
+        allowNull: false,
+        defaultValue: true,
```

Comment:
I think the defaultValue should be a string here. I think we would want to default it to maybe 'application_received' or something like that